Draft/draft-ietf-rtcweb-stun-consent-freshness.xml

```diff
@@ -233,13 +236,19 @@
       target="I-D.ietf-avtcore-srtp-ekt">EKT</xref>. Thus, in such shared
       keying distributions, receipt of an authenticated SRTP packet is not
       sufficient to verify consent.</t>
+
+      <t>The mechanism proposed in the document is an optional extension to
+      the ICE protocol, it can be deployed at one end of the two-party
+      communication session without impact on the other party.</t>
     </section>
 
     <section title="Solution">
       <t>Initial consent to send traffic is obtained using <xref
       target="RFC5245">ICE</xref>. An endpoint gains consent to send on a
-      candidate pair when the pair enters Succeeded ICE state. consent; this
-      document establishes a 30 second expiry time on consent.</t>
+      candidate pair when the pair enters Succeeded ICE state. This document
+      establishes a 30 second expiry time on consent. 30 second timeout period
+      is selected so that consent checks could be sent between 5 to 7 times to
+      handle packet loss.</t>
```

Comment:
David has asked to explicitly mention the below line in the draft.

-Tiru
From: Martin Thomson [mailto:notifications@github.com]
Sent: Thursday, May 28, 2015 9:46 PM
To: tireddy/STUNConsent
Cc: Tirumaleswar Reddy (tireddy)
Subject: Re: [STUNConsent] Opsdir and GenArt review (#1)


In Draft/draft-ietf-rtcweb-stun-consent-freshness.xml<https://github.com/tireddy/STUNConsent/pull/1#discussion_r31250644>:

>      </section>

>

>      <section title="Solution">

>        <t>Initial consent to send traffic is obtained using <xref

>        target="RFC5245">ICE</xref>. An endpoint gains consent to send on a

> -      candidate pair when the pair enters Succeeded ICE state. consent; this

> -      document establishes a 30 second expiry time on consent.</t>

> +      candidate pair when the pair enters Succeeded ICE state. This document

> +      establishes a 30 second expiry time on consent. 30 second timeout period

> +      is selected so that consent checks could be sent between 5 to 7 times to

> +      handle packet loss.</t>

This last sentence is unnecessary detail.

—
Reply to this email directly or view it on GitHub<https://github.com/tireddy/STUNConsent/pull/1/files#r31250644>.